src/lib/sidebar/EntireScheme.svelte

```diff
@@ -168,9 +181,22 @@
   <button
     type="button"
     class="align-right"
-    on:click={clearAll}
-    disabled={$gjScheme.features.length == 0}>Clear all</button
+    on:click={openClearAllDialogue}
+    disabled={$gjScheme.features.length == 0 || $isAToolInUse}>Clear all</button
   >
+  <ConfirmationModal
+    bind:open={displayClearAllConfirmation}
+    title={"Would you like to clear your work?"}
+    message={"This will delete all your drawn interventions."}
+    on:cancelAction={cancelClearAll}
+    on:confirmAction={clearAll}
+  />
+  {#if $isAToolInUse}
```

Comment:
![Screenshot from 2023-06-12 10-21-03](https://github.com/acteng/atip/assets/1664407/8b4e34a4-c467-46c2-8f55-fe8cd3cf4916)
The new text appearing feels a little distracting to me. What if we put tooltips on the disabled buttons instead, so if the user tries to interact with those functions and can't, they understand? Something like `title={$isAToolInUse ? "edit attribs first" : null}` seems to work. (I dunno if setting to `null` is the cleanest approach. Or how hard plumbing to `FileInput` will be)

Reply:
Should be easy to plumb to `FileInput`, should be simply adding a `disabledTooltip` prop. That said idk how many people would hover over a disabled button to work out why it's disabled. Maybe this is a question for a  UX type person?

Reply:
Indeed. I don't have a strong UX opinion, so let's keep it like this until we have more guidance or hear feedback. Thanks!